src/storybook/components/related-components/descriptions/checkbox-description.jsx

```diff
@@ -0,0 +1,14 @@
+import { useMemo } from "react";
+import { RelatedComponent } from "../../related-component/related-component";
+import { Checkbox } from "../../../../components/Checkbox/Checkbox";
+
+export const CheckboxDescription = () => {
+  const component = useMemo(() => <Checkbox label="Selected" checked={true} />, []);
```

Comment:
please use shorthand syntax for set a true (boolean) value to a prop, such in the following example: `<Checkbox label="Selected" checked />,`